src/test/resources/same/functions/functions.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+function1() {
+    echo "notUsingArgs1"
+}
+
+function function2() {
+    echo "notUsingArgs2"
+}
+
+function3() {
+    echo "arg = $1"
+}
+
+function4() {
+    echo "args = $1, $2"
+}
+
+function function5() {
+    echo "args = $1, $2, $3"
+}
+
+function3 "param"
```

Comment:
What kind of function is this? My bash isn't great. Got a link or something?

Reply:
@BFergerson My Bash also isn't great, so these are test calls to functions above them so that I am sure that they are working.

Reply:
Ah, okay. I couldn't understand how those could be functions but based on your comment they aren't functions. Please remove them from this file. It should only be function definitions. There shouldn't be any function calls.

Reply:
@BFergerson Please see my latest commit https://github.com/CodeBrig/Arthur/pull/95/commits/dfed9dd0fda4bd0179c0cba1ccd87122f0b54b76.